lti_consumer/lti_consumer.py

```diff
@@ -860,3 +860,15 @@ def _get_modal_position_offset(self, viewport_percentage):
             float: The css position offset to apply to the modal window
         """
         return (100 - viewport_percentage) / 2
+
+    def get_outcome_service_url(self, service_name="grade_handler"):
+        """
+        This function is called by get_course_lti_endpoints when using LTI result service to
+        discover the LTI result endpoints. It is similar to outcome_service_url property but
+        with a mapping for backward compatibility.
+        """
+        mapping = {
+            'grade_handler': 'outcome_service_handler',
+            'lti_2_0_result_rest_handler': 'result_service_handler'
+        }
+        return self.runtime.handler_url(self, mapping[service_name], thirdparty=True).rstrip('/?')
```

Comment:
Would it make sense to add a `result_service_url` call and instead of mapping to the service name, map to the function itself?  This would remove a little code redundancy, take advantage of the comments in the `outcome_service_url` property, and would have a little more consistency around the implementation.  Thoughts?